exe/import.cc

```diff
@@ -81,17 +81,20 @@ int main(int ac, char** av) {
     return 0;
   }
 
-  auto input_files = std::vector<std::pair<std::string, loader_config>>{};
+  auto input_files = std::vector<timetable_source>{};
   if (is_directory(in) && recursive) {
     for (auto const& e : fs::directory_iterator(in)) {
       if (is_directory(e) /* unpacked zip file */ ||
           boost::algorithm::to_lower_copy(
               e.path().extension().generic_string()) == ".zip") {
-        input_files.emplace_back(e.path().generic_string(), c);
+        input_files.emplace_back(
```

Comment:
if emplace_back doesn't really have any advantage over push_back like it's the case here (I think), it would make more sense to use push_back (less template instantiations = faster compilation + smaller binary size)